sccp/figures/figureThomsonDrop.py

```diff
@@ -124,14 +128,14 @@ def plotDifferentialExpression(
     axes[0].axline((0, b), slope=a, linestyle="--")
     axes[0].scatter([], [], c="b", label="Other Conditions")
     axes[0].scatter([], [], c="r", label=condition)
-    axes[0].set_xlabel(f"Component {most_exp_cmp + 1} Weight (Full Data)")
-    axes[0].set_ylabel(f"Component {most_exp_cmp + 1} Weight (Sampled Data)")
+    axes[0].set_xlabel(f"{cell_type} Component Weight (Full Data)")
+    axes[0].set_ylabel(f"{cell_type} Component Weight (Sampled Data)")
```

Comment:
Since the component numbers don't usually line up between the sampled and original data it is best to just refer to it as the "X Cell type component" instead of try and display two different component numbers and explain that